### ISSUE TITLE
IntelliSense doesn't work on my PC if I include a header there is defined std::string using CMake extensition

### ISSUE THREAD

**Issue by wusikijeronii** [microsoft/vscode-cpptools#10120]
### Environment

- OS and Version: Windows 11 22H2
- VS Code Version: 1.72,2
- C/C++ Extension Version: 1.12.4 (also tried pre-release version)
- Other extensions you installed (and if the issue persists after disabling them):  CMake v0.0.17
- If using SSH remote, specify OS of remote machine:  
- A clear and concise description of what the bug is, including information about the workspace (i.e. is the workspace a single project or multiple projects, size of the project, etc).

Hello.
I use VS Code on two machines: my laptop and my PC. On the laptop, everything works. On the PC, if the [CMake extension](https://marketplace.visualstudio.com/items?itemName=twxs.cmake) is enabled and I include a header with the `std::string` variable, IntelliSense stops working.

### Bug Summary and Steps to Reproduce

Bug Summary:
So, I've created a simple project with two files. You can find the source code below. On my laptop, it works. On the PC, it doesn't work. If I disable the CMake extension, IntelliSense successfully works. But my extensions sync, and I have the same extensions on different machines.

### Code sample and Logs

Project code:
main.cpp
```c++
#include <iostream>
#include "test.hpp"

std:: /// IntelliSense doesn't work
```

test.hpp
```c++
#ifndef TEST_HPP
#define TEST_HPP

#include <string>

std::string a{"test"};
#endif
```
settings.json (in project):
```json
{
    "files.associations": {
        "__bit_reference": "cpp",
        "__bits": "cpp",
        "__config": "cpp",
        "__debug": "cpp",
        "__errc": "cpp",
        "__hash_table": "cpp",
        "__locale": "cpp",
        "__mutex_base": "cpp",
        "__node_handle": "cpp",
        "__split_buffer": "cpp",
        "__threading_support": "cpp",
        "__tuple": "cpp",
        "__verbose_abort": "cpp",
        "array": "cpp",
        "atomic": "cpp",
        "bit": "cpp",
        "bitset": "cpp",
        "cctype": "cpp",
        "clocale": "cpp",
        "cmath": "cpp",
        "cstdarg": "cpp",
        "cstddef": "cpp",
        "cstdint": "cpp",
        "cstdio": "cpp",
        "cstdlib": "cpp",
        "cstring": "cpp",
        "ctime": "cpp",
        "cwchar": "cpp",
        "cwctype": "cpp",
        "deque": "cpp",
        "exception": "cpp",
        "initializer_list": "cpp",
        "ios": "cpp",
        "iosfwd": "cpp",
        "iostream": "cpp",
        "istream": "cpp",
        "limits": "cpp",
        "locale": "cpp",
        "memory": "cpp",
        "mutex": "cpp",
        "new": "cpp",
        "optional": "cpp",
        "ostream": "cpp",
        "ratio": "cpp",
        "stack": "cpp",
        "stdexcept": "cpp",
        "streambuf": "cpp",
        "string": "cpp",
        "string_view": "cpp",
        "system_error": "cpp",
        "tuple": "cpp",
        "type_traits": "cpp",
        "typeinfo": "cpp",
        "unordered_map": "cpp",
        "variant": "cpp",
        "vector": "cpp",
        "algorithm": "cpp"
    }
}
```
Debug logs
```
  other: --clang
  other: --clang_version=150003
  stdver: c++20
  intelliSenseMode: windows-clang-x64
Checking for syntax errors: D:\Documents\programming\istest\main.cpp
Queueing IntelliSense update for files in translation unit of: D:\Documents\programming\istest\main.cpp
cpptools/finishUpdateSquiggles
Update IntelliSense time (sec): 2.663
textDocument/didChange: D:\Documents\programming\istest\main.cpp
cpptools/textEditorSelectionChange
textDocument/didChange: D:\Documents\programming\istest\main.cpp
cpptools/textEditorSelectionChange
textDocument/completion: D:\Documents\programming\istest\main.cpp (id: 7)
auto_complete::handle_completion: file:///d%3A/Documents/programming/istest/main.cpp (4:6)
Offering completion
sending 1 changes to server
sending 1 changes to server
cpptools/getCodeActions: D:\Documents\programming\istest\main.cpp (id: 8)
cpptools/getFoldingRanges: D:\Documents\programming\istest\main.cpp (id: 9)
cpptools/getInlayHints: D:\Documents\programming\istest\main.cpp (id: 10)
cpptools/getDocumentSymbols: D:\Documents\programming\istest\main.cpp (id: 11)
cpptools/getDocumentSymbols
Checking for syntax errors: D:\Documents\programming\istest\main.cpp
IntelliSense process crash detected.
Update IntelliSense time (sec): 2.219
  tag parsing file: D:\Documents\programming\istest\main.cpp
Checking for syntax errors: D:\Documents\programming\istest\main.cpp
Update IntelliSense time (sec): 0
```
On error mode I just get:
```
IntelliSense process crash detected.
IntelliSense process crash detected.
IntelliSense process crash detected.
IntelliSense process crash detected.
IntelliSense process crash detected.
```

**Comment by wusikijeronii** [comment]
I removed default extension Make tools and it worked. But on my laptop the extension is installed and everything works. It's strange but my problem is solved

**Comment by wusikijeronii** [comment]
Same issue on another machine but removing make extension didn't help. But now it works if I remove [Cmake Tools extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cmake-tools).

**Comment by sean-mcmanus** [comment]
It looks like the `string` headers are leading to the crash, so we need more info on exactly which `string` headers are being used. Can you comment out the `std::string` usage so it doesn't crash and run the C/C++: Log Diagnostics command to the compilerPath and system Includes that are being used?

**Comment by wusikijeronii** [comment]
Thank you for the reply. I've commented lines included the `string`, then restarted the VS code and I got the following log:
```
loggingLevel: Debug
cpptools/didChangeCppProperties
Attempting to get defaults from C compiler in "compilerPath" property: 'C:\msys64\clang64\bin\gcc.exe'
Compiler query command line: C:\msys64\clang64\bin\gcc.exe -std=c17 -m64 -Wp,-v -fno-blocks -E -dM -x c nul
Code browsing service initialized
Attempting to get defaults from C++ compiler in "compilerPath" property: 'C:\msys64\clang64\bin\gcc.exe'
Compiler query command line: C:\msys64\clang64\bin\gcc.exe -std=c++14 -m64 -Wp,-v -fno-blocks -E -dM -x c++ nul
  Folder: C:/MSYS64/CLANG64/INCLUDE/ will be indexed
  Folder: C:/MSYS64/CLANG64/LIB/CLANG/15.0.3/INCLUDE/ will be indexed
  Folder: C:/USERS/ALEKESEJ/DOCUMENTS/PROGRAMMING/ISTEST/ will be indexed
cpptools/didChangeSettings
Discovering files...
IntelliSense Engine = Default.
Enhanced Colorization is enabled.
Error squiggles are enabled if all header dependencies are resolved.
Autocomplete is enabled.
textDocument/didOpen: C:\Users\alekesej\Documents\Programming\istest\main.cpp
cpptools/textEditorSelectionChange
textDocument/didChange: C:\Users\alekesej\Documents\Programming\istest\main.cpp
cpptools/textEditorSelectionChange
cpptools/activeDocumentChange: C:\Users\alekesej\Documents\Programming\istest\main.cpp
  tag parsing file: C:\Users\alekesej\Documents\Programming\istest\main.cpp
cpptools/activeDocumentChange: C:\Users\alekesej\Documents\Programming\istest\main.cpp
cpptools/getDocumentSymbols: C:\Users\alekesej\Documents\Programming\istest\main.cpp (id: 2)
cpptools/getInlayHints: C:\Users\alekesej\Documents\Programming\istest\main.cpp (id: 3)
cpptools/getDocumentSymbols
cpptools/getFoldingRanges: C:\Users\alekesej\Documents\Programming\istest\main.cpp (id: 4)
Request canceled: 2
cpptools/getDocumentSymbols: C:\Users\alekesej\Documents\Programming\istest\main.cpp (id: 5)
cpptools/getDocumentSymbols
cpptools/getSemanticTokens: C:\Users\alekesej\Documents\Programming\istest\main.cpp (id: 6)
  Processing folder (recursive): C:/MSYS64/CLANG64/INCLUDE/
Checking for syntax errors: C:\Users\alekesej\Documents\Programming\istest\main.cpp
  Processing folder (recursive): C:/MSYS64/CLANG64/LIB/CLANG/15.0.3/INCLUDE/
  Processing folder (recursive): C:/USERS/ALEKESEJ/DOCUMENTS/PROGRAMMING/ISTEST/
  Discovering files: 6864 file(s) processed
  0 file(s) removed from database
Done discovering files.
Populating include completion cache.
Parsing remaining files...
sending compilation args for C:\Users\alekesej\Documents\Programming\istest\main.cpp
  include: C:\MSYS64\CLANG64\INCLUDE\C++\V1
  include: C:\MSYS64\CLANG64\LIB\CLANG\15.0.3\INCLUDE
  include: C:\MSYS64\CLANG64\INCLUDE
  define: _WIN32=1
  define: _WIN64=1
  define: __ATOMIC_ACQUIRE=2
  define: __ATOMIC_ACQ_REL=4
  define: __ATOMIC_CONSUME=1
  define: __ATOMIC_RELAXED=0
  define: __ATOMIC_RELEASE=3
  define: __ATOMIC_SEQ_CST=5
  define: __BIGGEST_ALIGNMENT__=16
  define: __BITINT_MAXWIDTH__=128
  define: __BOOL_WIDTH__=8
  define: __BYTE_ORDER__=__ORDER_LITTLE_ENDIAN__
  define: __CHAR16_TYPE__=unsigned short
  define: __CHAR32_TYPE__=unsigned int
  define: __CHAR_BIT__=8
  define: __CLANG_ATOMIC_BOOL_LOCK_FREE=2
  define: __CLANG_ATOMIC_CHAR16_T_LOCK_FREE=2
  define: __CLANG_ATOMIC_CHAR32_T_LOCK_FREE=2
  define: __CLANG_ATOMIC_CHAR_LOCK_FREE=2
  define: __CLANG_ATOMIC_INT_LOCK_FREE=2
  define: __CLANG_ATOMIC_LLONG_LOCK_FREE=2
  define: __CLANG_ATOMIC_LONG_LOCK_FREE=2
  define: __CLANG_ATOMIC_POINTER_LOCK_FREE=2
  define: __CLANG_ATOMIC_SHORT_LOCK_FREE=2
  define: __CLANG_ATOMIC_WCHAR_T_LOCK_FREE=2
  define: __CONSTANT_CFSTRINGS__=1
  define: __DBL_DECIMAL_DIG__=17
  define: __DBL_DENORM_MIN__=4.9406564584124654e-324
  define: __DBL_DIG__=15
  define: __DBL_EPSILON__=2.2204460492503131e-16
  define: __DBL_HAS_DENORM__=1
  define: __DBL_HAS_INFINITY__=1
  define: __DBL_HAS_QUIET_NAN__=1
  define: __DBL_MANT_DIG__=53
  define: __DBL_MAX_10_EXP__=308
  define: __DBL_MAX_EXP__=1024
  define: __DBL_MAX__=1.7976931348623157e+308
  define: __DBL_MIN_10_EXP__=(-307)
  define: __DBL_MIN_EXP__=(-1021)
  define: __DBL_MIN__=2.2250738585072014e-308
  define: __DECIMAL_DIG__=__LDBL_DECIMAL_DIG__
  define: __DEPRECATED=1
  define: __EXCEPTIONS=1
  define: __FINITE_MATH_ONLY__=0
  define: __FLT16_DECIMAL_DIG__=5
  define: __FLT16_DENORM_MIN__=5.9604644775390625e-8F16
  define: __FLT16_DIG__=3
  define: __FLT16_EPSILON__=9.765625e-4F16
  define: __FLT16_HAS_DENORM__=1
  define: __FLT16_HAS_INFINITY__=1
  define: __FLT16_HAS_QUIET_NAN__=1
  define: __FLT16_MANT_DIG__=11
  define: __FLT16_MAX_10_EXP__=4
  define: __FLT16_MAX_EXP__=16
  define: __FLT16_MAX__=6.5504e+4F16
  define: __FLT16_MIN_10_EXP__=(-4)
  define: __FLT16_MIN_EXP__=(-13)
  define: __FLT16_MIN__=6.103515625e-5F16
  define: __FLT_DECIMAL_DIG__=9
  define: __FLT_DENORM_MIN__=1.40129846e-45F
  define: __FLT_DIG__=6
  define: __FLT_EPSILON__=1.19209290e-7F
  define: __FLT_HAS_DENORM__=1
  define: __FLT_HAS_INFINITY__=1
  define: __FLT_HAS_QUIET_NAN__=1
  define: __FLT_MANT_DIG__=24
  define: __FLT_MAX_10_EXP__=38
  define: __FLT_MAX_EXP__=128
  define: __FLT_MAX__=3.40282347e+38F
  define: __FLT_MIN_10_EXP__=(-37)
  define: __FLT_MIN_EXP__=(-125)
  define: __FLT_MIN__=1.17549435e-38F
  define: __FLT_RADIX__=2
  define: __FXSR__=1
  define: __GCC_ASM_FLAG_OUTPUTS__=1
  define: __GCC_ATOMIC_BOOL_LOCK_FREE=2
  define: __GCC_ATOMIC_CHAR16_T_LOCK_FREE=2
  define: __GCC_ATOMIC_CHAR32_T_LOCK_FREE=2
  define: __GCC_ATOMIC_CHAR_LOCK_FREE=2
  define: __GCC_ATOMIC_INT_LOCK_FREE=2
  define: __GCC_ATOMIC_LLONG_LOCK_FREE=2
  define: __GCC_ATOMIC_LONG_LOCK_FREE=2
  define: __GCC_ATOMIC_POINTER_LOCK_FREE=2
  define: __GCC_ATOMIC_SHORT_LOCK_FREE=2
  define: __GCC_ATOMIC_TEST_AND_SET_TRUEVAL=1
  define: __GCC_ATOMIC_WCHAR_T_LOCK_FREE=2
  define: __GCC_HAVE_SYNC_COMPARE_AND_SWAP_1=1
  define: __GCC_HAVE_SYNC_COMPARE_AND_SWAP_2=1
  define: __GCC_HAVE_SYNC_COMPARE_AND_SWAP_4=1
  define: __GCC_HAVE_SYNC_COMPARE_AND_SWAP_8=1
  define: __GNUC_GNU_INLINE__=1
  define: __GNUC_MINOR__=2
  define: __GNUC_PATCHLEVEL__=1
  define: __GNUC__=4
  define: __GNUG__=4
  define: __GXX_ABI_VERSION=1002
  define: __GXX_EXPERIMENTAL_CXX0X__=1
  define: __GXX_RTTI=1
  define: __GXX_WEAK__=1
  define: __INT16_C_SUFFIX__=
  define: __INT16_FMTd__="hd"
  define: __INT16_FMTi__="hi"
  define: __INT16_MAX__=32767
  define: __INT16_TYPE__=short
  define: __INT32_C_SUFFIX__=
  define: __INT32_FMTd__="d"
  define: __INT32_FMTi__="i"
  define: __INT32_MAX__=2147483647
  define: __INT32_TYPE__=int
  define: __INT64_C_SUFFIX__=LL
  define: __INT64_FMTd__="lld"
  define: __INT64_FMTi__="lli"
  define: __INT64_MAX__=9223372036854775807LL
  define: __INT64_TYPE__=long long int
  define: __INT8_C_SUFFIX__=
  define: __INT8_FMTd__="hhd"
  define: __INT8_FMTi__="hhi"
  define: __INT8_MAX__=127
  define: __INT8_TYPE__=signed char
  define: __INTMAX_C_SUFFIX__=LL
  define: __INTMAX_FMTd__="lld"
  define: __INTMAX_FMTi__="lli"
  define: __INTMAX_MAX__=9223372036854775807LL
  define: __INTMAX_TYPE__=long long int
  define: __INTMAX_WIDTH__=64
  define: __INTPTR_FMTd__="lld"
  define: __INTPTR_FMTi__="lli"
  define: __INTPTR_MAX__=9223372036854775807LL
  define: __INTPTR_TYPE__=long long int
  define: __INTPTR_WIDTH__=64
  define: __INT_FAST16_FMTd__="hd"
  define: __INT_FAST16_FMTi__="hi"
  define: __INT_FAST16_MAX__=32767
  define: __INT_FAST16_TYPE__=short
  define: __INT_FAST16_WIDTH__=16
  define: __INT_FAST32_FMTd__="d"
  define: __INT_FAST32_FMTi__="i"
  define: __INT_FAST32_MAX__=2147483647
  define: __INT_FAST32_TYPE__=int
  define: __INT_FAST32_WIDTH__=32
  define: __INT_FAST64_FMTd__="lld"
  define: __INT_FAST64_FMTi__="lli"
  define: __INT_FAST64_MAX__=9223372036854775807LL
  define: __INT_FAST64_TYPE__=long long int
  define: __INT_FAST64_WIDTH__=64
  define: __INT_FAST8_FMTd__="hhd"
  define: __INT_FAST8_FMTi__="hhi"
  define: __INT_FAST8_MAX__=127
  define: __INT_FAST8_TYPE__=signed char
  define: __INT_FAST8_WIDTH__=8
  define: __INT_LEAST16_FMTd__="hd"
  define: __INT_LEAST16_FMTi__="hi"
  define: __INT_LEAST16_MAX__=32767
  define: __INT_LEAST16_TYPE__=short
  define: __INT_LEAST16_WIDTH__=16
  define: __INT_LEAST32_FMTd__="d"
  define: __INT_LEAST32_FMTi__="i"
  define: __INT_LEAST32_MAX__=2147483647
  define: __INT_LEAST32_TYPE__=int
  define: __INT_LEAST32_WIDTH__=32
  define: __INT_LEAST64_FMTd__="lld"
  define: __INT_LEAST64_FMTi__="lli"
  define: __INT_LEAST64_MAX__=9223372036854775807LL
  define: __INT_LEAST64_TYPE__=long long int
  define: __INT_LEAST64_WIDTH__=64
  define: __INT_LEAST8_FMTd__="hhd"
  define: __INT_LEAST8_FMTi__="hhi"
  define: __INT_LEAST8_MAX__=127
  define: __INT_LEAST8_TYPE__=signed char
  define: __INT_LEAST8_WIDTH__=8
  define: __INT_MAX__=2147483647
  define: __INT_WIDTH__=32
  define: __LDBL_DECIMAL_DIG__=21
  define: __LDBL_DENORM_MIN__=3.64519953188247460253e-4951L
  define: __LDBL_DIG__=18
  define: __LDBL_EPSILON__=1.08420217248550443401e-19L
  define: __LDBL_HAS_DENORM__=1
  define: __LDBL_HAS_INFINITY__=1
  define: __LDBL_HAS_QUIET_NAN__=1
  define: __LDBL_MANT_DIG__=64
  define: __LDBL_MAX_10_EXP__=4932
  define: __LDBL_MAX_EXP__=16384
  define: __LDBL_MAX__=1.18973149535723176502e+4932L
  define: __LDBL_MIN_10_EXP__=(-4931)
  define: __LDBL_MIN_EXP__=(-16381)
  define: __LDBL_MIN__=3.36210314311209350626e-4932L
  define: __LITTLE_ENDIAN__=1
  define: __LLONG_WIDTH__=64
  define: __LONG_LONG_MAX__=9223372036854775807LL
  define: __LONG_MAX__=2147483647L
  define: __LONG_WIDTH__=32
  define: __MINGW32__=1
  define: __MINGW64__=1
  define: __MMX__=1
  define: __MSVCRT__=1
  define: __NO_INLINE__=1
  define: __NO_MATH_INLINES=1
  define: __OBJC_BOOL_IS_BOOL=0
  define: __OPENCL_MEMORY_SCOPE_ALL_SVM_DEVICES=3
  define: __OPENCL_MEMORY_SCOPE_DEVICE=2
  define: __OPENCL_MEMORY_SCOPE_SUB_GROUP=4
  define: __OPENCL_MEMORY_SCOPE_WORK_GROUP=1
  define: __OPENCL_MEMORY_SCOPE_WORK_ITEM=0
  define: __ORDER_BIG_ENDIAN__=4321
  define: __ORDER_LITTLE_ENDIAN__=1234
  define: __ORDER_PDP_ENDIAN__=3412
  define: __PIC__=2
  define: __POINTER_WIDTH__=64
  define: __PRAGMA_REDEFINE_EXTNAME=1
  define: __PTRDIFF_FMTd__="lld"
  define: __PTRDIFF_FMTi__="lli"
  define: __PTRDIFF_MAX__=9223372036854775807LL
  define: __PTRDIFF_TYPE__=long long int
  define: __PTRDIFF_WIDTH__=64
  define: __REGISTER_PREFIX__=
  define: __SCHAR_MAX__=127
  define: __SEG_FS=1
  define: __SEG_GS=1
  define: __SEH__=1
  define: __SHRT_MAX__=32767
  define: __SHRT_WIDTH__=16
  define: __SIG_ATOMIC_MAX__=2147483647
  define: __SIG_ATOMIC_WIDTH__=32
  define: __SIZEOF_DOUBLE__=8
  define: __SIZEOF_FLOAT128__=16
  define: __SIZEOF_FLOAT__=4
  define: __SIZEOF_INT128__=16
  define: __SIZEOF_INT__=4
  define: __SIZEOF_LONG_DOUBLE__=16
  define: __SIZEOF_LONG_LONG__=8
  define: __SIZEOF_LONG__=4
  define: __SIZEOF_POINTER__=8
  define: __SIZEOF_PTRDIFF_T__=8
  define: __SIZEOF_SHORT__=2
  define: __SIZEOF_SIZE_T__=8
  define: __SIZEOF_WCHAR_T__=2
  define: __SIZEOF_WINT_T__=2
  define: __SIZE_FMTX__="llX"
  define: __SIZE_FMTo__="llo"
  define: __SIZE_FMTu__="llu"
  define: __SIZE_FMTx__="llx"
  define: __SIZE_MAX__=18446744073709551615ULL
  define: __SIZE_TYPE__=long long unsigned int
  define: __SIZE_WIDTH__=64
  define: __SSE2_MATH__=1
  define: __SSE2__=1
  define: __SSE_MATH__=1
  define: __SSE__=1
  define: __STDCPP_DEFAULT_NEW_ALIGNMENT__=16ULL
  define: __STDCPP_THREADS__=1
  define: __STDC_HOSTED__=1
  define: __STDC_UTF_16__=1
  define: __STDC_UTF_32__=1
  define: __STDC__=1
  define: __STRICT_ANSI__=1
  define: __UINT16_C_SUFFIX__=
  define: __UINT16_FMTX__="hX"
  define: __UINT16_FMTo__="ho"
  define: __UINT16_FMTu__="hu"
  define: __UINT16_FMTx__="hx"
  define: __UINT16_MAX__=65535
  define: __UINT16_TYPE__=unsigned short
  define: __UINT32_C_SUFFIX__=U
  define: __UINT32_FMTX__="X"
  define: __UINT32_FMTo__="o"
  define: __UINT32_FMTu__="u"
  define: __UINT32_FMTx__="x"
  define: __UINT32_MAX__=4294967295U
  define: __UINT32_TYPE__=unsigned int
  define: __UINT64_C_SUFFIX__=ULL
  define: __UINT64_FMTX__="llX"
  define: __UINT64_FMTo__="llo"
  define: __UINT64_FMTu__="llu"
  define: __UINT64_FMTx__="llx"
  define: __UINT64_MAX__=18446744073709551615ULL
  define: __UINT64_TYPE__=long long unsigned int
  define: __UINT8_C_SUFFIX__=
  define: __UINT8_FMTX__="hhX"
  define: __UINT8_FMTo__="hho"
  define: __UINT8_FMTu__="hhu"
  define: __UINT8_FMTx__="hhx"
  define: __UINT8_MAX__=255
  define: __UINT8_TYPE__=unsigned char
  define: __UINTMAX_C_SUFFIX__=ULL
  define: __UINTMAX_FMTX__="llX"
  define: __UINTMAX_FMTo__="llo"
  define: __UINTMAX_FMTu__="llu"
  define: __UINTMAX_FMTx__="llx"
  define: __UINTMAX_MAX__=18446744073709551615ULL
  define: __UINTMAX_TYPE__=long long unsigned int
  define: __UINTMAX_WIDTH__=64
  define: __UINTPTR_FMTX__="llX"
  define: __UINTPTR_FMTo__="llo"
  define: __UINTPTR_FMTu__="llu"
  define: __UINTPTR_FMTx__="llx"
  define: __UINTPTR_MAX__=18446744073709551615ULL
  define: __UINTPTR_TYPE__=long long unsigned int
  define: __UINTPTR_WIDTH__=64
  define: __UINT_FAST16_FMTX__="hX"
  define: __UINT_FAST16_FMTo__="ho"
  define: __UINT_FAST16_FMTu__="hu"
  define: __UINT_FAST16_FMTx__="hx"
  define: __UINT_FAST16_MAX__=65535
  define: __UINT_FAST16_TYPE__=unsigned short
  define: __UINT_FAST32_FMTX__="X"
  define: __UINT_FAST32_FMTo__="o"
  define: __UINT_FAST32_FMTu__="u"
  define: __UINT_FAST32_FMTx__="x"
  define: __UINT_FAST32_MAX__=4294967295U
  define: __UINT_FAST32_TYPE__=unsigned int
  define: __UINT_FAST64_FMTX__="llX"
  define: __UINT_FAST64_FMTo__="llo"
  define: __UINT_FAST64_FMTu__="llu"
  define: __UINT_FAST64_FMTx__="llx"
  define: __UINT_FAST64_MAX__=18446744073709551615ULL
  define: __UINT_FAST64_TYPE__=long long unsigned int
  define: __UINT_FAST8_FMTX__="hhX"
  define: __UINT_FAST8_FMTo__="hho"
  define: __UINT_FAST8_FMTu__="hhu"
  define: __UINT_FAST8_FMTx__="hhx"
  define: __UINT_FAST8_MAX__=255
  define: __UINT_FAST8_TYPE__=unsigned char
  define: __UINT_LEAST16_FMTX__="hX"
  define: __UINT_LEAST16_FMTo__="ho"
  define: __UINT_LEAST16_FMTu__="hu"
  define: __UINT_LEAST16_FMTx__="hx"
  define: __UINT_LEAST16_MAX__=65535
  define: __UINT_LEAST16_TYPE__=unsigned short
  define: __UINT_LEAST32_FMTX__="X"
  define: __UINT_LEAST32_FMTo__="o"
  define: __UINT_LEAST32_FMTu__="u"
  define: __UINT_LEAST32_FMTx__="x"
  define: __UINT_LEAST32_MAX__=4294967295U
  define: __UINT_LEAST32_TYPE__=unsigned int
  define: __UINT_LEAST64_FMTX__="llX"
  define: __UINT_LEAST64_FMTo__="llo"
  define: __UINT_LEAST64_FMTu__="llu"
  define: __UINT_LEAST64_FMTx__="llx"
  define: __UINT_LEAST64_MAX__=18446744073709551615ULL
  define: __UINT_LEAST64_TYPE__=long long unsigned int
  define: __UINT_LEAST8_FMTX__="hhX"
  define: __UINT_LEAST8_FMTo__="hho"
  define: __UINT_LEAST8_FMTu__="hhu"
  define: __UINT_LEAST8_FMTx__="hhx"
  define: __UINT_LEAST8_MAX__=255
  define: __UINT_LEAST8_TYPE__=unsigned char
  define: __USER_LABEL_PREFIX__=
  define: __VERSION__="Clang 15.0.3"
  define: __WCHAR_MAX__=65535
  define: __WCHAR_TYPE__=unsigned short
  define: __WCHAR_UNSIGNED__=1
  define: __WCHAR_WIDTH__=16
  define: __WIN32=1
  define: __WIN32__=1
  define: __WIN64=1
  define: __WIN64__=1
  define: __WINNT=1
  define: __WINNT__=1
  define: __WINT_MAX__=65535
  define: __WINT_TYPE__=unsigned short
  define: __WINT_UNSIGNED__=1
  define: __WINT_WIDTH__=16
  define: __amd64=1
  define: __amd64__=1
  define: __cdecl=__attribute__((__cdecl__))
  define: __clang__=1
  define: __clang_literal_encoding__="UTF-8"
  define: __clang_major__=15
  define: __clang_minor__=0
  define: __clang_patchlevel__=3
  define: __clang_version__="15.0.3 "
  define: __clang_wide_literal_encoding__="UTF-16"
  define: __code_model_small__=1
  define: __cplusplus=201402L
  define: __declspec(a)=__attribute__((a))
  define: __fastcall=__attribute__((__fastcall__))
  define: __k8=1
  define: __k8__=1
  define: __llvm__=1
  define: __pascal=__attribute__((__pascal__))
  define: __pic__=2
  define: __private_extern__=extern
  define: __seg_fs=__attribute__((address_space(257)))
  define: __seg_gs=__attribute__((address_space(256)))
  define: __stdcall=__attribute__((__stdcall__))
  define: __thiscall=__attribute__((__thiscall__))
  define: __tune_k8__=1
  define: __x86_64=1
  define: __x86_64__=1
  define: _cdecl=__attribute__((__cdecl__))
  define: _fastcall=__attribute__((__fastcall__))
  define: _pascal=__attribute__((__pascal__))
  define: _stdcall=__attribute__((__stdcall__))
  define: _thiscall=__attribute__((__thiscall__))
  define: _DEBUG
  define: UNICODE
  define: _UNICODE
  other: --clang
  other: --clang_version=150003
  stdver: c++14
  intelliSenseMode: windows-clang-x64
Checking for syntax errors: C:\Users\alekesej\Documents\Programming\istest\main.cpp
Queueing IntelliSense update for files in translation unit of: C:\Users\alekesej\Documents\Programming\istest\main.cpp
cpptools/finishUpdateSquiggles
Error squiggle count: 0
Update IntelliSense time (sec): 2.027
  Parsing: 1114 files(s) processed
Done parsing remaining files.
cpptools/getFoldingRanges: C:\Users\alekesej\Documents\Programming\istest\main.cpp (id: 7)
```

**Comment by sean-mcmanus** [comment]
How did you install clang 15?

**Comment by wusikijeronii** [comment]
I used the following command in MSYS2 terminal:
```
pacman -S mingw-w64-clang-x86_64-clang
```

The detail info about installed package you can find [here](https://packages.msys2.org/package/mingw-w64-clang-x86_64-clang?repo=clang64)

**Comment by sean-mcmanus** [comment]
Yeah, that's the command I ran before, but I ran it in the CLANG64 terminal instead of the MSYS2 terminal, so I got it now and reproed the crash with 1.12.4, but the bug is fixed already with 1.13.4 (or 1.13.2, or 1.13.2, repros with 1.13.1 or earlier).
